library/templates/compare-xml.yaml

```diff
@@ -0,0 +1,67 @@
+- job-template:
```

Comment:
IMHO: this job is related to stash and its name or location should reflect this fact.